tools/i18n/makepot.php

```diff
@@ -68,14 +68,14 @@ class MakePOT {
 		),
 		'cp-plugin' => array(
 			'description' => 'Translation of the ClassicPress plugin {name} {version} by {author}',
-			'msgid-bugs-address' => 'https://wordpress.org/support/plugin/{slug}',
+			'msgid-bugs-address' => 'https://forums.classicpress.net/plugin/{slug}',
```

Comment:
This change is incorrect as we do not have subforums for each plugin.

Reply:
Let's leave it pointing to WP for now - we're using their plugin directory, makes sense to leave support there.

Reply:
The description should probably still say WordPress too.